test_workflow.sh

```diff
@@ -64,44 +64,45 @@ docker-compose exec -T conjur conjurctl account create myConjurAccount > admin_d
 echo
 
 echo "Step 6: Connect the Conjur client to the Conjur server"
-docker container exec conjur_client conjur init -u conjur -a myConjurAccount
+# `echo "Y"` is used to accept the self-signed certificate
+echo "Y" | docker container exec -i conjur_client conjur init -u https://proxy -a myConjurAccount --self-signed
 echo
 
 announce "UNIT 2. Define Policy"
 
 echo "Step 1: Log in to Conjur as admin"
 admin_api_key="$(cat admin_data | awk '/API key for admin/{print $NF}' | tr -d '\r')"
-docker-compose exec -T client conjur authn login -u admin -p ${admin_api_key}
+docker-compose exec -T client conjur login -u admin -p ${admin_api_key}
 echo
 
 echo "Step 2: Load the Sample Policy"
-docker-compose exec -T client conjur policy load root policy/BotApp.yml > my_app_data
+docker-compose exec -T client conjur policy load -b root -f policy/BotApp.yml > my_app_data
 echo
 
 echo "Step 3: Log out of Conjur as admin"
-docker-compose exec -T client conjur authn logout
+docker-compose exec -T client conjur logout
 echo
 
 announce "UNIT 3. Store a Secret in Conjur"
 
 echo "Step 1: Log in as Dave"
 cat my_app_data | awk '/"api_key":/{print $NF}' | tr -d '"' > my_api_keys
-dave_api_key="$(cat my_api_keys | awk 'NR==1')"
-docker-compose exec -T client conjur authn login -u Dave@BotApp -p ${dave_api_key}
+dave_api_key="$(cat my_api_keys | awk 'NR==2')"
```

Comment:
Oh. What happened here? It seems we added a new line. I would say maybe let's address this in the CLI so that any existing automation that users might have doesn't break

Reply:
Weird, hopefully it's consistent ordering